src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,64 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    public static final String WHITESPACE = " ";
+    public static final String COMMA = ",";
+    public static final String BUY = "buy";
+    public static final String SUPPLY = "supply";
+    public static final String RESULT = "result";
+    public static final int INDEX_OF_ACTION = 0;
+    public static final int INDEX_OF_QUANTITY = 1;
+    private String[] stringsArray;
 
     public void getStatistic(String fromFileName, String toFileName) {
```

Comment:
![image](https://user-images.githubusercontent.com/20326858/207034052-3eff779a-ff86-4982-bc40-af94996e58ea.png)
make it like this:
```
    public void getStatistic(String fromFileName, String toFileName) {
        String dataFromFile = readFromFile(fromFileName); // or // List<String> dataFromFile = read...
        String report = createReport(dataFromFile);
        writeToFile(report, toFileName);
    }
```